app/v1/lifecycle/systems/endpoints.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, Path
+
+from app.data.systems import OS_DATA_MOCKED
+
+v1_router = APIRouter()
+
+
+@v1_router.get("/systems")
+async def get_systems():
+    systems = get_systems_data()
+
+    return sorted(systems, key=lambda d: (d["major"], d["minor"]))
+
+
+@v1_router.get("/systems/{major}")
+async def get_systems_major(major: int = Path(..., description="Major version number")):
+    systems = get_systems_data(major)
```

Comment:
This is a call to a sync function from an async function. I need to look into this more, but I think this blocks the event loop.

I _think_ the correct way to call sync code from within async code is to use the default event loop with a Future object.

```suggestion
    loop = asyncio.get_running_loop()
    systems = await loop.run_in_executor(None, get_systems_data(major))
```

This isn't a problem currently because the blocking sync code is _fast_. But for a real IO or CPU bound operation, this could cause perceptible performance degradation.